io-engine/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -909,6 +916,12 @@ impl<'n> Nexus<'n> {
             // Reset operation is allowed only when the Nexus is Open state
             NexusState::Open => {
                 *state = NexusState::Reconfiguring;
+                EventWithMeta::event(
```

Comment:
Don't think we need such fine details, state changing to showdown should be enough?

Reply:
Yes the events list is filled with many state change events as we are having event for every state change. Other state change events are not needed for nexus other than shutdown? Or should I just remove reconfiguring event?

Reply:
Removed nexus state change events for reconfiguring and open states.